queue/driver_test.go

```diff
@@ -1005,7 +1005,7 @@ func TestDriverNextJob(t *testing.T) {
 
 					checkDispatched(t, next.Status())
 				},
-				"NextIgnoresJobThatHasNotHitWaitUntilRequirementYet": func(ctx context.Context, t *testing.T, driver *mongoDriver) {
+				"IgnoresJobThatHasNotHitWaitUntilRequirementYet": func(ctx context.Context, t *testing.T, driver *mongoDriver) {
```

Comment:
could it make sense to test this with a driver with `CheckWaitUntil == false`?

Reply:
Sure, I added it.